flexget/plugins/input/from_telegram.py

```diff
@@ -0,0 +1,219 @@
+
+    Example:
+      token: <token>
+      only_new: yes
+      whitelist:
+        - username: <my_username>
+        - group: <my_group>
+        - fullname:
+            first: <my_first_name>
+            sur: <my_surname>
+      entry:
+        <field>: <regexp to match value>
+
+    Note: If not declared, title will be the message
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'token': {'type': 'string'},
+            'types' : one_or_more({'type': 'string', 'enum': ['private', 'group']}),
+            'whitelist': {
+                'type': 'array',
+                'minItems': 1,
+                'items': {
+                    'oneOf': [
+                        {
+                            'type': 'object',
+                            'properties': {'username': {'type': 'string'}},
+                            'required': ['username'],
+                            'additionalProperties': False,
+                        },
+                        {
+                            'type': 'object',
+                            'properties': {
+                                'fullname': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'first': {'type': 'string'},
+                                        'sur': {'type': 'string'},
+                                    },
+                                    'required': ['first', 'sur'],
+                                    'additionalProperties': False,
+                                }
+                            },
+                            'required': ['fullname'],
+                            'additionalProperties': False,
+                        },
+                        {
+                            'type': 'object',
+                            'properties': {'group': {'type': 'string'}},
+                            'required': ['group'],
+                            'additionalProperties': False,
+                        },
+                    ]
+                },
+            },
+
+            'only_new': {'type': 'boolean','default': True},
+            'entry': {
+                'type': 'object',
+                'properties': {
+                    'url': {'type': 'string', 'format': 'regex'},
+                    'title': {'type': 'string', 'format': 'regex'},
+                },
+                'additionalProperties': {'type': 'string', 'format': 'regex'},
+            },
+        },
+        'required': ['token'],
+        'additonalProperties': False,
+    }
+
+    @plugin.internet(logger)
+    def on_task_input(self, task, config):
+        #Load The Configs
+        token = config['token']
+        only_new = config['only_new']
+        entry_config = config.get('entry')
+        whitelist = config.get('whitelist',[])
+        types = config.get('types')
```

Comment:
you can set the default here:
```suggestion
        types = config.get('types', ['private', 'group'])
```

Reply:
Done!